kernel/kselftest.py

```diff
@@ -211,11 +214,11 @@ def ksmtest(self):
         self.test_list = ["-M", "-Z", "-N", "-U", "-C"]
         if os.path.exists(ksm_test_bin):
             os.chdir(ksm_test_dir)
-            if(self.test_type == "-H" or self.test_type == "-P"):
+            if (self.test_type == "-H" or self.test_type == "-P"):
```

Comment:
This can be rewritten as 
`if (self.test_type in ["-H", "-P"]):`

Reply:
@sacsant any update?

Reply:
yup, will push updated changes.